packages/backend-common/config.d.ts

```diff
@@ -72,10 +75,23 @@ export interface Config {
       /** Default database client to use */
       client: 'better-sqlite3' | 'sqlite3' | 'pg';
       /**
-       * Base database connection string or Knex object
+       * Base database connection string or Knex (PG) object
+       * PG object details come from: https://node-postgres.com/apis/client
        * @visibility secret
        */
-      connection: string | object;
+      connection:
+        | string
+        | Partial<{
+            /**
+             * Password that belongs to the Postgres User
```

Comment:
Not necessarily specifically the postgres user - it depends - in particular when using sqlite or mysql :)

Reply:
Yes, should have caught that when I started. I've made this more generic now 👍